Lesson32/Merve/src/components/Exercise7.js

```diff
@@ -0,0 +1,9 @@
+export const Exercise7 = ({ onClick, children }) => {
+    return (
+        <div className="parent" onClick={onClick}>
+            Parent Element
+            {children} {}
```

Comment:
```suggestion
            {children}
```

Reply:
Normally, the goal was to add all the logic directly in this component. Move the child component and click handler function here from the App.js. No need to pass anything as props at this point.